web.py

```diff
@@ -260,106 +267,36 @@ def preprocess_dataset(trainset_dir, exp_dir, sr, n_p):
 
 
 # but2.click(extract_f0,[gpus6,np7,f0method8,if_f0_3,trainset_dir4],[info2])
-def extract_f0_feature(gpus, n_p, f0method, if_f0, exp_dir, version19, gpus_rmvpe):
-    gpus = gpus.split("-")
-    os.makedirs("%s/logs/%s" % (now_dir, exp_dir), exist_ok=True)
-    f = open("%s/logs/%s/extract_f0_feature.log" % (now_dir, exp_dir), "w")
-    f.close()
+def extract_f0_feature(
+    gpus: str, n_p: int, f0method: str, if_f0: bool, exp_dir: str, version19: str
+):
+    gpus_list: List[str] = gpus.split("-")
+
+    log_dir = Path(now_dir, "logs", exp_dir)
+    log_dir.mkdir(parents=True, exist_ok=True)
+
+    log_path = log_dir.joinpath("extract_f0_feature.log")
+    log_path.touch()
+
+    # ugly way to clear the file
+    open(log_path, "w").close()
+
     if if_f0:
-        if f0method != "rmvpe_gpu":
```

Comment:
If it is OK to remove the gpu comparison, just remove the whole rmvpe_gpu and merge it into rmvpe.

Reply:
> If it is OK to remove the gpu comparison, just remove the whole rmvpe_gpu and merge it into rmvpe.

To dynamically just use GPU if it's available? Sounds like a good idea to me.
Will do that after i fix features not saving

Reply:
Did that in 0b95532